ggml/src/ggml-sycl/CMakeLists.txt

```diff
@@ -104,8 +104,9 @@ if (GGML_SYCL_GRAPH)
 endif()
 
 # Link against Intel oneMKL or oneMath
-if (GGML_SYCL_TARGET STREQUAL "INTEL")
-    # Intel devices use Intel oneMKL directly instead of oneMath to avoid the limitation of linking Intel oneMKL statically
+find_package(oneMath)
+if (WIN32 AND NOT oneMath_FOUND AND GGML_SYCL_TARGET STREQUAL "INTEL")
```

Comment:
This will fallback to using the runtime dispatcher for Intel devices which can affect performance. You can see how the compile time dispatcher used to be enabled in https://github.com/ggml-org/llama.cpp/pull/12192/commits/995aea3d8309f049894aaffbc98cd8e774477b3e#diff-123d14cf628b04694c9022d4210d80c14a29409095635150b944f8dfd1c70b37L120.